docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -848,3 +848,24 @@ When I find = `5` elements by `By.xpath(//script):a` and for each element do
 |When I set 'type' attribute value of the context element to the 'scenario' variable 'type'|
 |Then `${type}` is equal to `text/javascript`                                              |
 ----
+
+=== Browser Steps
+==== Set the browser command line arguments
+
+Allows to set command line arguments for browsers. https://peter.sh/experiments/chromium-command-line-switches/[List of supported commands for Chrome browser]
```

Comment:
need to document:
- browsers supporting CLI args
- add links to such browsers args
- the behaviour of the step in case of declared property configuring CLI args